xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml

```diff
@@ -42,50 +42,92 @@
+    }
   ],
   'url': 'https://www.youtube.com/embed/zX1Itoh3E68',
   'duration': '5 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard3.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard3.topic1'),
-    $services.localization.render('help.videos.videoCard3.topic2'),
-    $services.localization.render('help.videos.videoCard3.topic3')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/CreatingAPage',
+      'label': "$services.localization.render('help.videos.videoCard3.topic1')"
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/EditingAPage',
+      'label': "$services.localization.render('help.videos.videoCard3.topic2')"
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/KeyboardShortcuts#HViewMode',
+      'label': "$services.localization.render('help.videos.videoCard3.topic3')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/onenEu21FIk',
   'duration': '6 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard4.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard4.topic1'),
-    $services.localization.render('help.videos.videoCard4.topic2')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/Imports#HImportingOfficedocuments',
+      'label': "$services.localization.render('help.videos.videoCard4.topic1')"
+    },
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/TOC+Macro#HExample1:Simple',
+      'label': "$services.localization.render('help.videos.videoCard4.topic2')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/csQVSRMVclM',
   'duration': '4 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard5.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard5.topic1'),
-    $services.localization.render('help.videos.videoCard5.topic2')
+    {
+      'url': $xwiki.getURL('Help.Applications.WebHome'),
+      'label': "$services.localization.render('help.videos.videoCard5.topic1')"
+    },
+    {
+      'url':'https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/Tutorials/FAQTutorial/FAQTutorialAWM#HCreatingnewFAQentries',
+      'label':"$services.localization.render('help.videos.videoCard5.topic2')"
+    }
   ],
   'url': 'https://www.youtube.com/embed/Pv4jPCaU99g',
   'duration': '7 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard6.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard6.topic1'),
-    $services.localization.render('help.videos.videoCard6.topic2')
+    {
+      'url':'https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/Tutorials/CreatingExtensions/#HInstallinganExtension',
+      'label':"$services.localization.render('help.videos.videoCard6.topic1')"
```

Comment:
```suggestion
      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/Tutorials/CreatingExtensions/#HInstallinganExtension',
      'label': "$services.localization.render('help.videos.videoCard6.topic1')"
```

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2466/commits/ff76bbeae9f5b0f5eeb4dec94bd18c93e3e76cfe :+1: